### PR TITLE
Removed Code to Load .tga files in TacRef

### PR DESCRIPTION
**Pull request by Stewie23**
Removed the Code that trys to load .tga files in the TacRef since .tga files are not used anymore

### CODE REVIEW

File: src/ui/src/tacref/ui_tacref.cpp [hunk 1]
```diff
@@ -781,18 +781,7 @@ static void Load3dModel(Entity *ent)
...
-
-    gImageMgr->RemoveImage(TACREF_BITMAP_ID);
-    strcpy(file, filename);
-    strcat(file, ".tga");
-    gImageMgr->LoadImage(TACREF_BITMAP_ID, file, 0, 0);
-    btn->Refresh();
-    btn->SetImage(0, TACREF_BITMAP_ID);
-    btn->Refresh();
-}
+
```
Comment (vinnydiehl) [inline src/ui/src/tacref/ui_tacref.cpp]: Hey, thanks for the code. Could you remove some of the whitespace here so there is only one blank line in between these two methods? Thanks.

Once it's done you don't have to make it a separate commit, just `git commit -a --amend` to squash the change into the previous commit and then `git push force origin RemoveTGA` to force Git to overwrite this one on the server.

**Comment by Stewie23** [comment]
Ah Thx for the Git-commands, i might have been a bit early to commit anyways, removed some other stuff that seems to belong to the .tga stuff.

**Comment by vinnydiehl** [comment]
Can you squash all 4 of these into a single commit with the original message? [Here's how.](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) Thanks again- just trying to keep everything neat and tidy.

**Comment by Stewie23** [comment]
okay i think i got it :)

**Comment by vinnydiehl** [comment]
Sweet.

The change is looking good, do you think you could also get rid of the declaration for `btn`? It's on [line 788](https://github.com/Stewie23/freefalcon-central/blob/31c445825f85b248a6cd1ccd78d055f61dc571a4/src/ui/src/tacref/ui_tacref.cpp#L788), at the beginning of the `EntityToWindow()` definition. Delete the line, this will kill a warning.

You can `--amend` that to keep this pull request to one commit. After that, I think we're ready to :ship: it.

Your message is repeated 4 times, by the way, think you can get rid of the repetitions when you go to amend it? It'll let you change the message. Thanks again.

**Comment by vinnydiehl** [comment]
Thanks! :beer:
